### PR TITLE
docs: update allowlist urls

### PR DESCRIPTION
**Pull request by jaffrepaul**

### CODE REVIEW

File: docs/faq/questions/cloud-faq.mdx [hunk 1]
```diff
@@ -267,18 +267,19 @@ Cloud.
 - `https://assets.cypress.io` - **Asset CDN** (Org logos, icons, videos,
   screenshots, etc.)
 - `https://authenticate.cypress.io` - **Authentication API**
-- `https://capture.cypress.io` - **Cypress Test Replay**
```
Comment (jmherbst) [inline docs/faq/questions/cloud-faq.mdx:270]: Can we leave this line in place?

File: docs/faq/questions/cloud-faq.mdx [hunk 2]
```diff
@@ -267,18 +267,19 @@ Cloud.
...
 **If you are using GitHub Enterprise or GitLab for Enterprise (Self-managed),
-you may also need to add both the following to the version control IP allow
-list:**
+you may also need to add the following to the version control IP allowlist:**
 
 - `3.211.102.119` - **Dedicated IP**
 - `18.213.72.78` - **Dedicated IP**
+- `35.169.145.173` - **Dedicated IP**
+- `44.199.152.70` - **Dedicated IP**
+- `52.70.95.89` - **Dedicated IP**
```
Comment (jmherbst) [inline docs/faq/questions/cloud-faq.mdx:282]: These are correct, thank you 👍

File: docs/guides/references/troubleshooting.mdx [hunk 3]
```diff
@@ -228,6 +228,15 @@ Cloud.
 - `https://download.cypress.io` - **CDN download of Cypress binary**
 - `https://on.cypress.io` - **URL shortener for link redirects**
 
+**If you are using GitHub Enterprise or GitLab for Enterprise (Self-managed),
```
Comment (jmherbst) [inline docs/guides/references/troubleshooting.mdx:231]: Oh, I wasn't aware of this document originally I guess.  Thanks for adding the IPs here 🥳 

Could you also add the capture url above in this same file as well?
```
`https://capture.cypress.io` - **Cypress Test Replay**
```

**Review by jmherbst — CHANGES_REQUESTED** [review]

File: docs/faq/questions/cloud-faq.mdx (hunk 1 above)
Comment (jaffrepaul) [reply docs/faq/questions/cloud-faq.mdx:270]: This is being added when the v13 docs branch is deployed. It was added here incorrectly.

File: docs/guides/references/troubleshooting.mdx (hunk 3 above)
Comment (jaffrepaul) [reply docs/guides/references/troubleshooting.mdx:231]: same on this one

**Review by jmherbst — APPROVED** [review]

**Pull request merged**